inference-engine/src/preprocessing/ie_preprocess_data.hpp

```diff
@@ -38,12 +38,14 @@ class IPreProcessData : public details::IRelease {
      * @brief Sets ROI blob to be resized and placed to the default input blob during pre-processing.
      * @param blob ROI blob.
      */
+    //FIXME: rename to setUserBlob
     virtual void setRoiBlob(const Blob::Ptr &blob) = 0;
 
     /**
      * @brief Gets pointer to the ROI blob used for a given input.
      * @return Blob pointer.
      */
+    //FIXME: rename to getUserBlob
```

Comment:
let's do this renaming.

Reply:
I would prefer to do it in a separate activity, as most likely it will affect all plugins that do override  `SetBlob` and therefore use this interface